CHANGELOG.md

```diff
@@ -20,7 +20,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Added speed of light and Planck's constant to PhysConstants module
 - Added `GFED4_CLIMATOLOGY` option to HEMCO_Config.rc
 - Added CH4 emissions from hydroelectric reservoirs to CH4, Carbon, and tagCH4 simulations
-- Add RxnConst diagnostic for archiving reaction rate constants
+- Added RxnConst diagnostic for archiving reaction rate constants
+- Added new tracer_mod.F90 containing subroutines for applying sources and sinks for the TransportTracer simulation
```

Comment:
Were any new transport tracers added?

Reply:
Yes, a few new tracers were added in addition to renaming existing ones. This is now clarified in CHANGELOG.md.